src/utils.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import 'setimmediate';
 import validateFormData from './validate';
+import { isObject as _isObject } from 'lodash';
```

Comment:
why the alias?

Reply:
`isObject` is declared in this file, it's an already existing helper from RJSF.